cmd/kubeadm/app/cmd/join.go

```diff
@@ -162,14 +161,30 @@ func NewCmdJoin(out io.Writer, joinOptions *joinOptions) *cobra.Command {
 			c, err := joinRunner.InitData(args)
 			kubeadmutil.CheckErr(err)
 
+			data := c.(*joinData)
+			fmt.Printf("[join] Using Kubernetes version: %s\n", data.initCfg.KubernetesVersion)
```

Comment:
This is the line generating panic. Please remove it (this output does not add value IMO)

Reply:
removed it.